### PR TITLE
Support to handle multiple group tags in an xliff file

### PR DESCRIPTION
**Pull request by gouniLee**
The loctool need to support multiple groups in an xliff case.
Currently, loctool considers only one group tag in an xliff file. but we have a case to have two groups in xliff file. something like:
https://github.com/iLib-js/loctool/blob/xliffGroupArray/test/testfiles/test5.xliff

### CODE REVIEW

File: lib/Xliff.js [hunk 1]
```diff
@@ -1028,83 +1028,86 @@ Xliff.prototype.parse2 = function(xliff) {
             };
 
             unitsElement = file.group || file;
+            unitsElement = ilib.isArray(unitsElement) ? unitsElement : [unitsElement];
```
Comment (ehoogerbeets) [inline lib/Xliff.js]: That's what `makeArray()` does

File: test/testfiles/test5.xliff [hunk 2]
```diff
@@ -42,5 +42,16 @@
     </segment>
    </unit>
  </group>
+ <group id="settings_g2" name="x-qml">
+    <unit id="settings_22">
+    <notes>
+     <note>copy strings from voiceagnet</note>
```
Comment (ehoogerbeets) [inline test/testfiles/test5.xliff]: voiceagnet?

File: lib/Xliff.js [hunk 3]
```diff
@@ -1028,83 +1028,86 @@ Xliff.prototype.parse2 = function(xliff) {
...
-                    var source = "", target = "";
-
-                    if (tu.notes && tu.notes.note) {
-                        comment = ilib.isArray(tu.notes.note) ?
-                            tu.notes.note[0]["$t"] :
-                            tu.notes.note["$t"];
-                    }
+                    for (var k=0; k < transUnits.length; k++) {
+                        var comment = "", state;
+                        var datatype = transUnits[k]["l:datatype"] || unitsElement[j].name;
```
Comment (ehoogerbeets) [inline lib/Xliff.js]: `transUnits.forEach()` is slightly faster than a `for` loop.

File: docs/ReleaseNotes.md [hunk 4]
```diff
@@ -1,6 +1,15 @@
 Release Notes for Version 2
 ============================
 
+Build 008
+-------
+
+Published as version 2.4.1
```
Comment (ehoogerbeets) [inline docs/ReleaseNotes.md]: When adding a new feature, the minor version number should be increased. In this case, 2.5.0

Comment (gouniLee) [reply docs/ReleaseNotes.md]: Yes, you're right. I update it.

**Pull request merged**